bus-mapping/src/evm/opcodes/error_oog_sload_sstore.rs

```diff
@@ -0,0 +1,96 @@
+use super::{Opcode, OpcodeId};
+use crate::circuit_input_builder::{CircuitInputStateRef, ExecStep};
+use crate::operation::{CallContextField, RW};
+use crate::operation::{StorageOp, TxAccessListAccountStorageOp};
+use crate::Error;
+use eth_types::{GethExecStep, ToWord};
+
+/// Placeholder structure used to implement [`Opcode`] trait over it
+/// corresponding to the
+/// [`OogError::SloadSstore`](crate::error::OogError::SloadSstore).
+#[derive(Clone, Copy, Debug)]
+pub(crate) struct OOGSloadSstore;
+
+impl Opcode for OOGSloadSstore {
+    fn gen_associated_ops(
+        state: &mut CircuitInputStateRef,
+        geth_steps: &[GethExecStep],
+    ) -> Result<Vec<ExecStep>, Error> {
+        let geth_step = &geth_steps[0];
+        debug_assert!([OpcodeId::SLOAD, OpcodeId::SSTORE].contains(&geth_step.op));
+
+        let mut exec_step = state.new_step(geth_step)?;
+        let next_step = geth_steps.get(1);
+        exec_step.error = state.get_step_err(geth_step, next_step).unwrap();
```

Comment:
can assign targetting error directly `exec_step.error = ExecError::OutOfGas(OogError::SloadSstore) `

Reply:
Fixed in commit [2833dd4](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1147/commits/2833dd445a76d2c2fca19ee9f089c9601948ef46).